platzky_sendmail/entrypoint.py

```
@@ -1,30 +1,75 @@
+import logging
 import smtplib
 from typing_extensions import Annotated
```

Ruff (0.14.13)

Import from `typing` instead: `Annotated`

Import from `typing`

(UP035)